Dockerfile

```diff
@@ -1,54 +1,48 @@
-FROM google/debian:wheezy
-MAINTAINER u@gogs.io
-
-RUN echo "deb http://ftp.debian.org/debian/ wheezy-backports main" >> /etc/apt/sources.list && \
-	apt-get update -qqy && \
-	apt-get install --no-install-recommends -qqy \
-	curl build-essential ca-certificates git \ 
-	openssh-server libpam-dev && \
-	apt-get autoclean && \
-    apt-get autoremove && \
-    rm -rf /var/lib/apt/lists/*
-
-ENV GOROOT /goroot
-ENV GOPATH /gopath
-ENV PATH $PATH:$GOROOT/bin:$GOPATH/bin
-
-COPY . /gopath/src/github.com/gogits/gogs/
-WORKDIR /gopath/src/github.com/gogits/gogs/
+FROM debian:jessie
```

Comment:
Yes. The ENV variables still remain in docker although are unneeded cause GOROOT and GOPATH will be deleted.

I think that could keep COPY line (`COPY . /gopath/src/github.com/gogits/gogs/`), cause the Docker image are build with DockerHub, in this case we can remove the git clone lines (16 and 17):
> git clone https://github.com/gogits/gogs.git \
>    /gopath/src/github.com/gogits/gogs && \

And can remove the `mkdir /gopath/src/github.com/gogits` in 14 cause the copy do the magic.